### PR TITLE
RT-2.8: IS-IS Passive is enabled at the area level

### PR DESCRIPTION
**Pull request by divyarema**
This resolves https://github.com/openconfig/featureprofiles/issues/1616

### CODE REVIEW

File: feature/experimental/isis/ate_tests/isis_interface_level_passive_test/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+## Procedure
+
+* Configure IS-IS for ATE port-1 and DUT port-1.
+* Configure DUT interface with IS-IS passive configured at area level 2. 
+* Verify that IS-IS adjacency is not coming up in level-2 area for IPv4 and IPV6 address families.
+* Verify that IS-IS adjacency for IPv4 and IPV6 address families is coming up in level-1 area.
+* Verify that IPv4 and IPv6 prefixes that are advertised by ATE correctly installed into DUTs route and forwarding table.
+* Ensure that IPv4 and IPv6 prefixes that are advertised as part of an (emulated) neighboring system are installed into the DUT routing table, and validate that packets are sent and received to them.
+* TODO-Verify the output of ST path displaying the interface as passive in ISIS database/adj table
```
Comment (sachendras) [inline feature/experimental/isis/ate_tests/isis_interface_level_passive_test/README.md:15]: Why is this a todo?

**Review by sachendras — APPROVED** [review]

File: feature/experimental/isis/ate_tests/isis_interface_level_passive_test/README.md (hunk 1 above)
Comment (divyarema) [reply feature/experimental/isis/ate_tests/isis_interface_level_passive_test/README.md:15]: Passive can be verified under isis interface or isis interface level hierarchy using 
interfaces/interface/state/passive or interfaces/interface/levels/level/state/passive.
However not clear which is the ST path displaying the interface as passive in ISIS database/adj table.

**Review by arulkumarsekar — APPROVED** [review]

File: feature/experimental/isis/otg_tests/isis_interface_level_passive_test/README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+# RT-2.8: IS-IS Passive is enabled at the area level
```
Comment (sachendras) [inline feature/experimental/isis/otg_tests/isis_interface_level_passive_test/README.md:1]: Sorry of the miscommunication here. Requirement here is just for IS-IS level 2 area and IS-IS level1 not needed. Therefore lets keep the test simple and as below


# RT-2.8: IS-IS Passive is enabled at the area level

## Summary

* Verify isis adjacency with passive enabled under level.

## Procedure

- Configure IS-IS for ATE port-1 and DUT port-1.
- Configure DUT interface with IS-IS passive configured at area level 2. 
	- Verify that IS-IS adjacency is not coming up in level-2 area for IPv4 and IPV6 address families.
- Undo the IS-IS passive configuration under level 2
	- Verify that IS-IS adjacency for IPv4 and IPV6 address families are coming up in the level-2 area.
	- Verify that IPv4 and IPv6 prefixes that are advertised by ATE are correctly installed into DUTs route and forwarding table.
	- Ensure that IPv4 and IPv6 prefixes that are advertised as part of an (emulated) neighboring system are installed into the DUT routing table, and validate that packets are sent and received to them.
	- TODO-Verify the output of ST path displaying the interface as passive in ISIS database/adj table

Comment (divyarema) [reply feature/experimental/isis/otg_tests/isis_interface_level_passive_test/README.md]: modified

**Review by arulkumarsekar — APPROVED** [review]

**Review by sachendras — APPROVED** [review]

**Review by greg-dennis — APPROVED** [review]
Approving for Go quality

**Pull request merged**